force_bdss/io/workflow_writer.py

```diff
@@ -1,12 +1,12 @@
 import json
-from traits.api import HasStrictTraits, Unicode
+from traits.api import HasStrictTraits, Str
 
 
 class WorkflowWriter(HasStrictTraits):
     """A Writer for writing the Workflow onto disk.
     """
 
-    version = Unicode('1')
+    version = Str("1")
```

Comment:
Nice, this is something we can look out for in upcoming PRs - I will make an issue to remind us

Reply:
Thank you, I guess the `Unicode` trait was needed to handle `python2.*` versions in the first place. Which is not supported anymore.